test/Microsoft.AspNet.Mvc.DataAnnotations.Test/RangeAttributeAdapterTest.cs

```diff
@@ -18,6 +20,15 @@ public void GetClientValidationRules_ReturnsValidationParameters()
             var metadata = provider.GetMetadataForProperty(typeof(string), "Length");
 
             var attribute = new RangeAttribute(typeof(decimal), "0", "100");
+            attribute.ErrorMessage = "The field Length must be between {1} and {2}.";
+
+            var expectedProperties = new object[] { "0", "100" };
+            var expectedMessage = "The field Length must be between 0 and 100.";
+
+            var stringLocalizer = new Mock<IStringLocalizer>();
+            stringLocalizer.Setup(s => s[attribute.ErrorMessage, expectedProperties])
+                .Returns(new LocalizedString(attribute.ErrorMessage, expectedMessage));
```

Comment:
If the localizer returns a `string` that lacks format parameters, this test isn't showing things work correctly / completely. Looks like a general issue in these new tests.